go.mod

```diff
@@ -9,15 +9,16 @@ require (
 	github.com/google/addlicense v0.0.0-20190907113143-be125746c2c4
 	github.com/open-telemetry/opentelemetry-collector v0.2.1-0.20191209163440-5d463fe48816
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/azuremonitorexporter v0.0.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sapmexporter v0.0.0-00010101000000-000000000000
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/signalfxexporter v0.0.0-20191203211755-8ae89debd6c5
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/stackdriverexporter v0.0.0-20191126142441-b2a048090ad6
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/collectdreceiver v0.0.0-20191209163404-28d5712f4129
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sapmreceiver v0.0.0-20191209183045-9707a2732f37
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinscribereceiver v0.0.0-20191126142441-b2a048090ad6
-	github.com/open-telemetry/opentelemetry-collector-contrib/testbed v0.0.0-20191209183045-9707a2732f37 // indirect
 	github.com/pavius/impi v0.0.0-20180302134524-c1cbdcb8df2b
-	github.com/pierrec/lz4 v2.0.5+incompatible // indirect
 	golang.org/x/lint v0.0.0-20191125180803-fdd1cda4f05f
 	golang.org/x/tools v0.0.0-20191205225056-3393d29bb9fe
 	honnef.co/go/tools v0.0.1-2019.2.3
 )
+
+replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sapmexporter => ./exporter/sapmexporter
```

Comment:
nit: can we have all replaces here on the bottom? there is a previous one on the top